package.json

```diff
@@ -12,8 +12,8 @@
     "start": "node --es-module-specifier-resolution=node ./build/index.js",
     "dev": "NODE_ENV=\"development\" nodemon index.ts",
     "test": "mocha ./src/**/*.test.*",
-    "docker": "COMPOSE_PROJECT_NAME=inversify-template docker-compose up",
-    "docker-rebuild": "docker container stop inversify-template_backend_1 inversify-template_pgadmin_1 inversify-template-db; docker system prune && docker volume prune -f --filter \"label=com.docker.compose.project=inversify-template\" && COMPOSE_PROJECT_NAME=inversify-template docker-compose up --force-recreate --build",
+    "docker": "COMPOSE_PROJECT_NAME=inversify-template docker compose up",
+    "docker-rebuild": "docker container stop inversify-template_backend_1 inversify-template_pgadmin_1 inversify-template-db; docker system prune && docker volume prune -f --filter \"label=com.docker.compose.project=inversify-template\" && COMPOSE_PROJECT_NAME=inversify-template docker compose up --force-recreate --build",
```

Comment:
"docker-compose" didn't work on my Ubuntu, but "docker compose" worked. Looking back I think I should've created a new script for Ubuntu so it doesn't complicate anything on your end.